util/topgen/top.py

```diff
@@ -4,7 +4,7 @@
 
 '''Code representing the entire chip for reggen'''
 
-from typing import Dict, List, Optional, Tuple, Union
+from typing import Dict, List, Optional, Tuple, Set
```

Comment:
Union is used in line 71, so keep it. Also, the list of imports is sorted alphabetically, so reorder Set.

Reply:
I agree that it is used as a typing hint. Though when importing `Union`, CI fails with:
```
  flake8 util/design/gen-top-docs.py util/reggen/gen_dv.py util/tlgen/elaborate.py util/tlgen/item.py util/tlgen/lib.py util/tlgen/validate.py util/tlgen/xbar.py util/topgen.py util/topgen/c.py util/topgen/c_test.py util/topgen/gen_dv.py util/topgen/intermodule.py util/topgen/lib.py util/topgen/merge.py util/topgen/rust.py util/topgen/top.py util/topgen/validate.py
	util/topgen/top.py:7:1: F401 'typing.Union' imported but unused
```

Reply:
My mistake: Union is used within a comment, so should not be imported.